WORKSPACE.bazel

```diff
@@ -7,7 +7,7 @@ local_repository(
 
 load("//cuda:repositories.bzl", "register_detected_cuda_toolchains", "rules_cuda_dependencies")
 
-rules_cuda_dependencies()
+rules_cuda_dependencies("11.8")
```

Comment:
These can't be hardcoded as this breaks the CI tests of different versions of CUDA